framework/interface/templates/base.html

```diff
@@ -120,6 +120,31 @@
     function disableInput(selector) {
         $(selector).prop("disabled", true);
     }
+
+    // Function that fills the recently completed targets
+    function RecentlyFinisedTargets() {
```

Comment:
When the function updates, I can visually confirm the change (the dropdown shakes a little bit) - every 3s (3000ms). IMO this should not happen. Use timeouts to only fill the array from the API call i.e I mean ** use the timeouts on the API call - not the whole div rendering function. 

So instead of this: 
`$.getJSON('/api/targets/recent/` use `.$.ajax`